packages/babel-helpers/src/helpers.js

```diff
@@ -426,12 +426,21 @@ helpers.objectDestructuringEmpty = template(`
 `);
 
 helpers.objectWithoutProperties = template(`
-  (function (obj, keys) {
+  (function (source, excluded) {
     var target = {};
-    for (var i in obj) {
-      if (keys.indexOf(i) >= 0) continue;
-      if (!Object.prototype.hasOwnProperty.call(obj, i)) continue;
-      target[i] = obj[i];
+    var sourceKeys = source == null? [] : Object.keys(source);
+    
+    if (Object.getOwnPropertySymbols) {
+      sourceKeys = sourceKeys.concat(Object.getOwnPropertySymbols(source));
+    }
+
+    var key;
+    for (var i = 0; i < sourceKeys.length; i++) {
+      key = sourceKeys[i];
+      if (!Object.prototype.propertyIsEnumerable.call(source, key)) continue;
+      if (excluded.indexOf(key) >= 0) continue;
+      if (!Object.prototype.hasOwnProperty.call(source, key)) continue;
```

Comment:
This should no longer be necessary (`Object.keys` returns own keys only, and so does `Object.getOwnPropertySymbols`)